data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,495 @@ mission "Unfettered returning home"
+				goto "try again"
+	on defer
+		set "Hai Legacy 1: deferred"
+	on accept
+		clear "unfettered: ocean dialog"
+		clear "unfettered: pilot dialog"
+
+mission "Hai Legacy 2"
+	autosave
+	landing
+	name "Heritage reclaimed"
+	description "Use the newly acquired Anomalocaris to bring aid to the Unfettered Hai on <planet> by <date>, with it as your flagship and preferably without any other ships."
+	clearance
+	# deadline to make sure you don't just enjoy free rep
+	deadline 30
+	cargo "food (aid)" 121
+	source "Hai-home"
+	destination "Firelode"
+	to offer
+		not "Hai Legacy 1: declined"
+		has "First Contact: Unfettered: offered"
+		"reputation: Hai (Unfettered)" < 0
+		not "Unfettered Jump Drive 1: offered"
+		"credits" >= 10000000
+	to accept
+		has "ship model: Anomalocaris"
+	on offer
+		conversation
+			`Upon returning to the spaceport, one could say the cargo finds you more than you find it. A worker immediately greets you while others begin to fill your ship with crates, which a quick check shows have already been allocated for this ship in your name. All that is left to do is to bring it to <destination> by <date> for the promised sum of <payment>.`
+			choice
+				`	(Prepare for takeoff.)`
+			`	As you head to the bridge, preparing to take off from the planet, you notice a message on the command panel, from your "associates," probably:`
+			`	"Use the Anomalocaris as your flagship.`
+			`	"Do not bring any other ships, as they are not needed for the cargo and would only be a liability at best in Unfettered space, when they could easily provoke them at worst."`
+			`	Given the potential dangers of that area of space, it would be a good idea to familiarize yourself with the design of the Anomalocaris and even customize it more to your own liking - especially if you plan to deliver the cargo in only this single ship, as requested.`
+				accept
+	on enter
+		system
+			attributes "unfettered"
+		conversation
+			branch "wrong flagship"
+				not "flagship model: Anomalocaris"
+			branch "not single ship"
+				"total ships" != 1
+			action
+				"old reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)"
+				"reputation: Hai (Unfettered)" >?= 0
+			`As you enter the system, you prepare for a fight to get the cargo through, but it is surprisingly quiet. For the first time, entering Unfettered space does not mean getting shot at by missiles from all sides. They're not hailing you either. Perhaps they think you're one of the Hai aiding them, especially given your cargo and the ship you're flying.`
+				accept
+			label "not single ship"
+			`You enter the system, and, for a brief moment, nobody is shooting you.`
+			`	The Unfettered quickly change their mind, however, as soon as your escorts enter the system. They happily head your way, weapons ready to loot your cargo. It becomes clear you should have come with only the Anomalocaris, as the associates suggested.`
```

Comment:
Perhaps this is fine, but seems a bit weird if your escorts are several systems out and you are shot anyway. Maybe change the text to say that the Unfettered may have information that you have several ships somehow.

Reply:
Since they will be hostile anyways, maybe "The Unfettered must have information on the several ships in your fleet, as they seem more than happy to engage you as you enter the system. It becomes clear...."

Reply:
the "total ships" condition only applies to ships in the current system, actually